plugins/org.python.pydev.parser/src/org/python/pydev/parser/visitors/NodeUtils.java

```diff
@@ -1549,9 +1549,11 @@ public static TypeInfo getTypeForClassDefAttribute(String attributeWithoutSelf,
             if (stmtType instanceof Assign) {
                 Assign assign = (Assign) stmtType;
                 if (assign.type != null) {
-                    String representationString = getRepresentationString(assign.type);
-                    if (attributeWithoutSelf.equals(representationString)) {
-                        return new TypeInfo(assign.value);
+                    for (exprType target : assign.targets) {
+                        String representationString = getRepresentationString(target);
```

Comment:
Actually, the block below already does the checking for the target, so, you can already use it (remember that an assign to multiple targets can't really have typing info, so, it checks if len == 1 already).

Something as:

```
if (attributeWithoutSelf.equals(getRepresentationString(assign.targets[0]))) {
    if (assign.type != null) {
        return new TypeInfo(assign.type);
    }
```